aimon/decorators/analyze.py

```diff
@@ -16,6 +18,133 @@ def __init__(self, name, model_type, metadata={}):
         self.model_type = model_type
         self.metadata = metadata
 
+class EvalResponse:
+    def __init__(self, output, response):
+        self.output = output
+        self.response = response
+
+    def __str__(self):
+        return f"EvalResponse(output={self.output}, response={self.response})"
+
+    def __repr__(self):
+        return str(self)
+
+
+def run_eval(
+        application_name,
+        model_name,
+        dataset_collection_name, 
+        evaluation_name, 
+        headers, 
+        api_key=None,
+        aimon_client=None,
+        eval_tags=None, 
```

Comment:
eval_tags is not used in the code

Reply:
yes, not right now. It will be good to remove it.